device-types/Cisco/WS-C3850-12X48U-S.yaml

```diff
@@ -0,0 +1,190 @@
+---
+manufacturer: Cisco
+model: Catalyst 3850-12X48U-S
+slug: cisco-ws-c3850-12x48u-s
+part_number: WS-C3850-12X48U-S
```

Comment:
So funny enough, the 3850 actually makes this distinction in inventory, even though the device itself is soft-upgradeable to a higher license level I believe.

```
NAME: "Switch 1", DESCR: "WS-C3850-48F-L"
PID: WS-C3850-48F-L    , VID: V07  , SN: <REDACTED>
```